backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidApplication.java

```diff
@@ -186,6 +191,15 @@ protected void createWakeLock (boolean use) {
 		}
 	}
 
+	@TargetApi(28)
```

Comment:
Maybe use `Build.VERSION_CODES.P` here as well to see they match?

Reply:
I followed the used format in `useImmersiveMode`.